_zh-cn/overviews/scala3-book/ca-given-imports.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 给定导入
+type: section
+description: This page demonstrates how 'given' import statements work in Scala 3.
+num: 62
+previous-page: ca-context-bounds
+next-page: ca-extension-methods
+---
+
+
+为了更清楚地说明当前作用域中的给定来自何处，我们使用一种特殊形式的 `import` 语句来导入 `given` 实例。
```

Comment:
Please, don't translate given 
The following is also.

Reply:
changed.